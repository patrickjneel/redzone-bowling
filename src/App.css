.App {
  text-align: center;
}


.App-header {
  align-items: center;
  background-color: #222;
  display: flex;
  height: 124px;
  justify-content: center;
  padding: 20px;
  color: white;
}

.App-title {
  color: white;
  font-family: 'Roboto', sans-serif;;
  font-size: 50px;
  margin: unset;
  padding-top: 7px;
}

.logo-img {
  width: 385px;
}

