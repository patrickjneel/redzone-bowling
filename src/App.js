import React, { Component } from 'react';
import './App.css';
import BowlingArea from './components/BowlingArea';

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <img className="logo-img" src={require('./redzone.png')} />
          <h1 className="App-title">BOWLING</h1>
        </header>
        <BowlingArea />
      </div>
    );
  }
}

export default App;
