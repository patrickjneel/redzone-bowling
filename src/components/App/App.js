import React, { Component } from 'react';
import './App.css';
import BowlingArea from '../BowlingArea/BowlingArea';

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <img className="logo-img" src={require('../../assets/redzone.png')} alt="redzone-logo"/>
          <h1 className="App-title">BOWLING</h1>
        </header>
        <img className="bowling-img" src={require('../../assets/bowl.jpg')} />
        <BowlingArea />
      </div>
    );
  }
}

export default App;
