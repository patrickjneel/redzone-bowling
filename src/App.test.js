import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

it('should match the snap shot', () => {
    const renderedApp = shallow(<App />);
    expect(renderedApp).toMatchSnapshot();

  });
