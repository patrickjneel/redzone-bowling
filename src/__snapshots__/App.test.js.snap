// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snap shot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header
          className="App-header"
        >
          <img
            alt="redzone-logo"
            className="logo-img"
            src="redzone.png"
          />
          <h1
            className="App-title"
          >
            BOWLING
          </h1>
        </header>,
        <img
          className="bowling-img"
          src="bowl.jpg"
        />,
        <BowlingArea />,
      ],
      "className": "App",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="redzone-logo"
              className="logo-img"
              src="redzone.png"
            />,
            <h1
              className="App-title"
            >
              BOWLING
            </h1>,
          ],
          "className": "App-header",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "redzone-logo",
              "className": "logo-img",
              "src": "redzone.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "BOWLING",
              "className": "App-title",
            },
            "ref": null,
            "rendered": "BOWLING",
            "type": "h1",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "bowling-img",
          "src": "bowl.jpg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="App-header"
          >
            <img
              alt="redzone-logo"
              className="logo-img"
              src="redzone.png"
            />
            <h1
              className="App-title"
            >
              BOWLING
            </h1>
          </header>,
          <img
            className="bowling-img"
            src="bowl.jpg"
          />,
          <BowlingArea />,
        ],
        "className": "App",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="redzone-logo"
                className="logo-img"
                src="redzone.png"
              />,
              <h1
                className="App-title"
              >
                BOWLING
              </h1>,
            ],
            "className": "App-header",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "redzone-logo",
                "className": "logo-img",
                "src": "redzone.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "BOWLING",
                "className": "App-title",
              },
              "ref": null,
              "rendered": "BOWLING",
              "type": "h1",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "bowling-img",
            "src": "bowl.jpg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
